fjord/feedback/tests/test_views.py

```diff
@@ -176,12 +176,6 @@ def test_happy_prefill_in_querystring_is_ignored(self):
 
     def test_firefox_os_view(self):
         """Firefox OS returns correct view"""
-        # Firefox OS is the user agent
-        url = reverse('feedback')
-        ua = 'Mozilla/5.0 (Mobile; rv:18.0) Gecko/18.0 Firefox/18.0'
-        r = self.client.get(url, HTTP_USER_AGENT=ua)
-        assert template_used(r, 'feedback/fxos_feedback.html')
```

Comment:
I ditched this feature. Basically, if you go to `/feedback/` with a Firefox OS phone, it would guess that you were using Firefox OS based on the user agent and then you'd get the Firefox OS feedback form. That's a terrible idea.

Now, it'll send you to the product picker and you can pick Firefox OS. It won't affect many people since Firefox 1.3+ uses an in-product feedback form and not the one hosted on Input.